scripts/benchmarks.py

```diff
@@ -109,6 +109,17 @@ def accept(seed, testdata, value):
     return accept
 
 
+def lower_bound(seed, testdata, value):
+    rnd = random.Random(seed)
+    for b in testdata.buffer:
+        c = rnd.randint(0, 255)
+        if c < b:
+            return True
+        if c > b:
+            return False
+    return True
```

Comment:
It's the standard interface that literally every benchmark uses and it's entirely contained within this file, so I'm not sure adding a documentation line to each one of them saying what the arguments are is particularly useful.